config/environments/test.rb

```diff
@@ -46,7 +46,7 @@
   config.action_mailer.default_options = {
     from: "mail@example.com",
   }
-  config.domain = "localhost:3000"
+  config.domain = "www.example.com"
```

Comment:
What's the significance of changing this?

Reply:
It makes links in tests look more obviously 'test' like. Localhost links look much more like they belong in develop environment.